docs/ontologymodeling/metadata.md

```diff
@@ -1,3 +1,172 @@
+| VBO:0300007 | domestic breed | Domestication status given to a group of animals that has been domesticated, that is have been selectively bred and genetically adapted over generations to live alongside humans.               |
+| VBO:0300008 | wild breed     | Domestication status given to a group of animals that has not been domesticated and exists as free-living, self-sustaining and unmanaged population.                                             |
+
+As with all metadata, the source of information is recorded. Currently, the domestication status information solely comes from DAD-IS
+
+Examples:
+
+<img src="Screenshot 2024-07-31 at 3.41.53 PM.png" width="250">
+<img src="Screenshot 2024-07-31 at 3.42.25 PM.png" width="200">
+<img src="Screenshot 2024-07-31 at 3.42.56 PM.png" width="250">
+
+## Extinction status
+
+_upcoming_
+
+## Description of origin
+
+Information about the breed origin, including information about the foundation stock used to create a breed, is often available in the form of free text notes. The free text note format is not ideal for querying information, but since this information is useful to some users, we recorded it, as reported by the sources (i.e. in free text note format), using the annotation property “description of origin” (http://purl.obolibrary.org/obo/vbo#origindescription). As for all other metadata, the source for the information is included.
+
+It should be noted that 
+- ongoing work aims to review the ‘description of origin’ information and create ‘has_foundation_stock’ axioms to record this information in a more semantically useful format
+- some information beyond the description of origin is sometimes included. 
+
+<img src="Screenshot 2024-07-31 at 4.00.31 PM.png" width="500">
+
+## Comment
+
+Information on term concepts and usage are provided in the comment.
+
+Example: 
+
+<img src="Screenshot 2024-07-31 at 4.04.05 PM.png" width="500">
+
+
+## in subset - transboundary
+
+Users requested a way to identify VBO terms corresponding to DAD-IS transboundary breeds (read more about transboundary breeds and DAD-IS HERE). To support users of the DAD-IS data, we added a subset annotation to these VBO terms. 
+The annotation uses the “in subset” (http://www.geneontology.org/formats/oboInOwl#inSubset) annotation property, with the “transboundary” annotation. 
+
+Example: 
+
+<img src="Screenshot 2024-07-31 at 4.12.45 PM.png" width="300">
+
+## term tracker
+
+All ontology updates are tracked in our [GitHub repository](https://github.com/monarch-initiative/vertebrate-breed-ontology). When an update or edit on a VBO term is made in the context of a GitHub issue, a link to GitHub issue is added the VBO term. 
+These links are useful to keep track of editing history (e.g. what changes were made) and for transparency (i.e. one can go back to a GitHub issue to gain context on the discussions and decisions that lead to the changes in the VBO term)
```

Comment:
Add period at the end of the last sentence.

Reply:
done